src/tests/multihost/alltests/test_proxy_rfc2307.py

```diff
@@ -0,0 +1,671 @@
+        domain_params = {'case_sensitive': 'false',
+                         'access_provider': 'simple',
+                         'simple_deny_groups': 'user_cs1_grp1'
+                         }
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        for user in ['User_cs1', 'user_cs1_alias']:
+            execute_cmd(multihost, "> /var/log/secure")
+            client = pexpect_ssh(client_hostip, user, 'Secret123', debug=False)
+            with pytest.raises(Exception):
+                client.login(login_timeout=10, sync_multiplier=1,
+                             auto_prompt_reset=False)
+            time.sleep(3)
+            execute_cmd(multihost, f'cat /var/log/secure | grep -i "Access denied for user {user}"')
+        execute_cmd(multihost, "> /var/log/secure")
+        execute_cmd(multihost, "sed -i 's/user_cs1_grp1/user_cs1_grp1_alias/' /etc/sssd/sssd.conf")
+        tools.clear_sssd_cache()
+        for user in ['user_cs1', 'user_cs1_alias']:
+            client = pexpect_ssh(client_hostip, user, 'Secret123', debug=False)
+            try:
+                client.login(login_timeout=30, sync_multiplier=5,
+                             auto_prompt_reset=False)
+            except SSHLoginException:
+                pytest.fail("%s failed to login" % user)
+            else:
+                client.logout()
+
+    def test_outgoing_ldaps(self, multihost, backupsssdconf):
+        """
+        :title: proxy provider: outgoing ldaps and ldap ports rejected
+        :id: 15709f7e-43b6-11ed-9c3f-845cf3eff344
+        :steps:
+          1. cache_credentials=true
+          2. login with ldap user
+          3. Block outgoing ldaps
+          3. login with ldap user
+          4. Unblock outgoing ldaps
+          5. login with ldap user
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Should succeed
+          5. Should succeed
+        """
+        config_sssd(multihost)
+        tools = sssdTools(multihost.client[0])
+        sssd_params = {'domains': ds_instance_name}
+        tools.sssd_conf('sssd', sssd_params)
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'access_provider': 'simple', 'cache_credentials': "true"}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        client_hostip = multihost.client[0].ip
+        client = pexpect_ssh(client_hostip, "User_CS1", 'Secret123', debug=False)
+        try:
+            client.login(login_timeout=30, sync_multiplier=5,
+                         auto_prompt_reset=False)
+        except SSHLoginException:
+            pytest.fail("%s failed to login" % "User_CS1")
+        else:
+            client.logout()
+        host_ip = multihost.master[0].ip
+        # block master server
+        block_ip = multihost.client[0].run_command(f'iptables -I OUTPUT -d {host_ip} -j DROP')
```

Comment:
Hi,

why do you block the full host, the original test is only blocking ports 389 (ldap) and 636 (ldaps). This might be important if the same host is e.g. used for DNS during the test. You can either use two iptables commands as the original test or use `-p tcp --match multiport --dport 389,636`.

bye,
Sumit

Reply:
Done